### PR TITLE
Catalog Ingestion Feature Switcher

### PR DESCRIPTION
**Pull request by garmashdev**
# Description
Added 2 new feature switchers:

M2_CATALOG_INGESTION -> default disabled
M2_STORE_CONFIGURATION -> default enabled

Fixes: (link ticket)

#changelog Catalog Ingestion Feature Switcher

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please validate that you have tested your change in at least one of the following areas:

- [ ] Successfully tested locally (or docker image)
- [ ] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# For PR Reviewer 

- [ ] Reviewed unit tests to make sure we are using real components rather than mocks as much as possible?
- [ ] For any major change (observer, new Bolt feature, core Magento interaction) we must add a feature switch, did you verify this?

# Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have created or modified unit tests to sufficiently cover my changes.
- [ ] I have added my ticket link and provided a changelog message.

### CODE REVIEW

File: Helper/FeatureSwitch/Definitions.php [hunk 1]
```diff
@@ -566,5 +576,17 @@ class Definitions
             self::DEFAULT_VAL_KEY => false,
             self::ROLLOUT_KEY     => 100
         ],
+        self::M2_CATALOG_INGESTION => [
+            self::NAME_KEY        => self::M2_CATALOG_INGESTION,
+            self::VAL_KEY         => true,
+            self::DEFAULT_VAL_KEY => false,
+            self::ROLLOUT_KEY     => 100
```
Comment (vitaliyreznikov) [inline Helper/FeatureSwitch/Definitions.php:583]: should be 0

File: Helper/FeatureSwitch/Definitions.php [hunk 2]
```diff
@@ -566,5 +576,17 @@ class Definitions
...
+        ],
+        self::M2_STORE_CONFIGURATION => [
+            self::NAME_KEY        => self::M2_STORE_CONFIGURATION,
+            self::VAL_KEY         => true,
+            self::DEFAULT_VAL_KEY => true,
```
Comment (vitaliyreznikov) [inline Helper/FeatureSwitch/Definitions.php:588]: should be false

File: Helper/FeatureSwitch/Definitions.php [hunk 3]
```diff
@@ -277,6 +277,16 @@ class Definitions
...
 
+    /**
+     * Enable catalog ingestion
+     */
+    const M2_CATALOG_INGESTION = 'M2_CATALOG_INGESTION';
+
+    /**
+     * Enable store configuration update request
+     */
+    const M2_STORE_CONFIGURATION = 'M2_STORE_CONFIGURATION';
```
Comment (vitaliyreznikov) [inline Helper/FeatureSwitch/Definitions.php:288]: maybe M2_STORE_CONFIGURATION_WEBHOOK

Comment (garmashdev) [reply Helper/FeatureSwitch/Definitions.php]: Updated.

File: Helper/FeatureSwitch/Definitions.php (hunk 1 above)
Comment (garmashdev) [reply Helper/FeatureSwitch/Definitions.php]: Updated

**Review by vitaliyreznikov — APPROVED** [review]

**Pull request merged**